wandb/sdk/wandb_run.py

```diff
@@ -671,6 +671,15 @@ def _init(
                 os.path.join("code", self._settings.program_relpath)
             )
 
+        if self._settings.fork_from is not None:
+            # verify that the specified project matches the current project.
+            # forking from a different project or entity is not currently supported.
+
+            config[wandb_key]["branch_point"] = {
+                "run_id": self._settings.fork_from.run,
+                "step": self._settings.fork_from.value,
+            }
```

Comment:
it doesn't look like this is verifying anything like the comment says; instead it's mutating "branch_point"

Reply:
relic of something that was there before; removed